configs/experiment/m2/feat_locsConfs.yaml

```diff
@@ -0,0 +1,122 @@
+# @package _global_
+
+# to execute this experiment run:
+# python train.py experiment=example
+task: "m2"
+# feature_version: 6
+topic: "medical"
+
+defaults:
+  - override /data: ptg
+  - override /model: ptg
+  - override /callbacks: default
+  - override /trainer: gpu
+  - override /paths: default
+  #- override /logger: aim
+  - override /logger: csv
+
+# all parameters below will be merged with parameters from default configurations set above
+# this allows you to overwrite only specified parameters
+
+# Change this name to something descriptive and unique for this experiment.
+# This will differentiate the run logs and output to be separate from other
+# experiments that may have been run under the configured
+# Setting this value influences:
+# - the name of the directory under `${paths.root_dir}/logs/` in which training
+#   run files are stored.
+# Default is "train" set in the "configs/train.yaml" file.
+#task_name:
+task_name: cameron_locs_and_confs
```

Comment:
Probably leave these customizations out in what we check in.

Reply:
Removed